tests/unit/cli/test_data_loader.py

```diff
@@ -263,7 +263,7 @@ def test_load_single_file_parquet(tmpdir, dict_rows_input, expected_documents):
     path = tmpdir.join("test.parquet")
     pd.DataFrame(data).to_parquet(str(path))
 
-    docs = _load_single_file_by_suffix(str(path))
+    docs = _load_single_schematic_file_by_suffix(str(path))
     assert docs == expected_documents
 
 
```

Comment:
Can we please add at least one unit test for this use case? Especially a UT of a directory with mixed `.jsonl` and `.txt` files?